### ISSUE TITLE
AnnotationPage does not have id

### ISSUE THREAD

**Issue by jptmoore** [allmaps/allmaps#141]
Hi, thanks for the awesome work!

I notice when trying to parse your annotation pages they are missing an id. For example https://annotations.allmaps.org/manifests/c390af06ea724803

The Web Annotation Data Model says there must be one https://www.w3.org/TR/annotation-model/#annotation-page

**Comment by bertspaan** [comment]
Good point. I'll add an ID to all the AnnotationPages! It's an easy fix but it'll take me a little longer to deploy a new version of the Allmaps API, I'll do that probably within a week or 2.

**Comment by sammeltassen** [comment]
Seems fixed!
